dl4j-cv-labs/pom.xml

```diff
@@ -128,10 +128,18 @@
         <!-- For running opencv and ffmpeg libraries -->
         <dependency>
             <groupId>org.bytedeco</groupId>
-            <artifactId>javacv</artifactId>
+            <artifactId>javacv-platform</artifactId>
             <version>${javacv.version}</version>
         </dependency>
 
+        <!-- https://mvnrepository.com/artifact/org.bytedeco/javacv -->
+        <dependency>
+            <groupId>org.bytedeco.javacpp-presets</groupId>
+            <artifactId>ffmpeg-platform</artifactId>
+            <version>3.2.1-1.3</version>
```

Comment:
put the version in root pom.xml https://github.com/CertifaiAI/TrainingLabs/blob/chooneung_ffmpeg-issue/pom.xml#L38-L61

and refer to it in this line

Reply:
this is for easier management of multiple version in only changing them in one place rather than dig through it through all the pom.xml available.

Reply:
changes made: https://github.com/CertifaiAI/TrainingLabs/pull/115/commits/7375f0b273f4a56f9bcdd000ae3a667aff3855bd
Kindly review @codenamewei